src/main/java/knightminer/inspirations/common/IHidable.java

```diff
@@ -0,0 +1,25 @@
+package knightminer.inspirations.common;
+
+import knightminer.inspirations.Inspirations;
+import net.minecraft.item.ItemGroup;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Used for blocks/items that are disableable in the config.
+ */
+public interface IHidable {
+    boolean isEnabled();
```

Comment:
Should add javadocs for these method rather than inline comments, even if its just
```
/** Some description */
```